python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1335,8 +1358,23 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
 
             for d, detector in enumerate(visitSummary["id"]):
                 mapName = f"{visit}/{detector}"
-
-                mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                if mapName in wcsf.mapCollection.allMapNames():
+                    mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                else:
+                    # This extension was not fit, but try to recover WCS
+                    genericElements = mapTemplate["EXPOSURE/DEVICE/base"]["Elements"]
+                    mapElements = []
+                    instrument = visitSummary[0].getVisitInfo().getInstrumentLabel()
+                    for component in genericElements:
+                        elements = mapTemplate[component]["Elements"]
+                        for element in elements:
+                            for generic, specific in {
+                                "BAND": instrument,
```

Comment:
Is `instrument` really a thing to substitute for `band`?

Reply:
It probably shouldn't be, but these lines are trying to replicate how `gbdes` names mappings, and the instrument name is what ends up here (probably due to this line: https://github.com/lsst/gbdes/blob/4efbc7097dc222cb35d22f8aed79572413aa1a97/include/Instrument.h#L21). A near-term item on my to-do list is to switch to fitting multiple bands at once, so this will probably be addressed there.